extensions/extensions-helm/aissemble-spark-operator-chart/README.md

```diff
@@ -45,7 +45,6 @@ aissemble-spark-operator-chart:
 | volumes                    | Volumes for the pod                | No                | `spark-logging=/tmp/spark-logging`          |
 | volumeMounts               | Volume Mounts for the pod          | No                | `spark-logging=/tmp/spark-logging`          |
 | fullnameOverride           | String to override release name    | No                | spark-operator                              |
-| rbac.createClusterRole     | See `Migrated Properties`          | No                | false                                       |
```

Comment:
I: Custom RBAC doesn't seem to be needed anymore, and using ours caused failures with the latest Spark Operator version

Reply:
A: There is more README below this about our custom RBAC that should be cleaned up/ removed

Reply:
The first time I read that I thought it was giving v1 to v2 migration details and that the info was still accurate, except the new RBAC is provided by the base chart instead of our chart.  But on second read you're totally correct. Will remove.